.changes/breaking/2746.md

```diff
@@ -0,0 +1,22 @@
+Breaking changes to the CLI arguments:
+- To disable `random-walk` just don't specify it. Before it required to use `--random-walk 0`.  
```

Comment:
Looking at. the code before it was : `default_value = 0` which means no need to add it explicitly.